run-qa-checks

```diff
@@ -5,6 +5,14 @@ set -e
 echo ''
 echo 'Checking documentation build status'
 ./runsphinx-build
+
+# check for broken link
+python -m http.server 8001 -d docs/build/ &>/dev/null &
+pid=$!
+sleep 4
+pylinkvalidate.py http://localhost:8001/
+kill "${pid}"
+
```

Comment:
It works, finding the broken links in so many links is hard, so a QA is `+1` from me! :smile: